src/User/pages/Privacy-Policy/Privacy.jsx

```
@@ -328,56 +303,65 @@ const Privacy = () => {
-                </a>
-              </p>
-              <p>
-                <b>Time:</b> Mon - Fri (9:00 - 18:00)
-              </p>
-              <p>
-                <b>Address:</b> Kanpur , Uttar Pradesh
-              </p>
+              <p>In accordance with the Information Technology Act, 2000 and the rules made thereunder, the name and contact details of the Grievance Officer are provided below:</p>
+              <ul style={{ marginLeft: "20px", listStyleType: "none" }}>
+                <li><b>Name:</b> Vivek Prajapati</li>
+                <li><b>Email:</b> <a className="a" href="mailto:grievanceofficer@vigybag.com" target="_blank">grievanceofficer@vigybag.com</a></li>
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)